src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -293,10 +293,10 @@ class EthService(
         gasUsed = if (txIndex == 0) receipt.cumulativeGasUsed else receipt.cumulativeGasUsed - receipts(txIndex - 1).cumulativeGasUsed,
         contractAddress = contractAddress,
         logs = receipt.logs.zipWithIndex.map { case (txLog, index) =>
-          TxLog(
+          Log(
```

Comment:
I think it is I just picked Log because I did not want to change @LukasGasior1 code
but you are right, will change this